### ISSUE TITLE
"Cancel appointment" button accessibility issue

### ISSUE THREAD

**Issue by digitalken** [department-of-veterans-affairs/va.gov-team#20870]
## What happened?
In iOS and Android:
Cancel button is missing accessibilityHint.

Should be:
Cancel button

- Title= Cancel appointment
- accessibilityHint= “Cancel this appointment"
- accessibilityRole= “button"

## Specs:
See above.

## Steps to Reproduce
With screen reader software enabled, use phone to read the Cancel appointment button.

## Desired behavior
Cancel button

- Title= Cancel appointment
- accessibilityHint= “Cancel this appointment"
- accessibilityRole= “button"

## Acceptance Criteria
Cancel button

- Title= Cancel appointment
- accessibilityHint= “Cancel this appointment"
- accessibilityRole= “button"

## Bug Severity
- 4 - Low

## Linked to story
https://app.zenhub.com/workspaces/va-mobile-app-5f205eac3155a2000f422ecb/issues/department-of-veterans-affairs/va.gov-team/20457

## Screen shot(s) and additional information
n/a

**Comment by digitalken** [comment]
PASS - 3/4/2021 - Two bugs logged, however.  1)  Cancel appointment button had capitalized 'A', but shouldn't.  2) Accessibility, the Cancel button is missing the hint.

**Comment by digitalken** [comment]
PASS 3/9/2021 - Looks good on Android, iOS.  Closing.
